pkg/controllers/kube-apiserver.go

```diff
@@ -125,7 +125,14 @@ func (s *KubeAPIServer) configure(cfg *config.Config) error {
 			"kubelet-certificate-authority":   {cryptomaterial.CABundlePath(kubeCSRSignerDir)},
 			"kubelet-client-certificate":      {cryptomaterial.ClientCertPath(kubeletClientDir)},
 			"kubelet-client-key":              {cryptomaterial.ClientKeyPath(kubeletClientDir)},
-			"kubelet-preferred-address-types": {"InternalIP", "Hostname"},
+			// MicroShift nodes expose these two types of addresses. In order to support having more than one
```

Comment:
The current approach (using standalone kubelet) does not share certs with the master node, it uses a cert with cluster-admin privileges to bootstrap: https://kubernetes.io/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/

Reply:
In light of this I will remove the second commit, as its handled correctly by bootstrapping.